system/lib/libcxxabi/src/cxa_exception_emscripten.cpp

```diff
@@ -30,121 +39,116 @@ __gxx_personality_v0(int version,
                      _Unwind_Context* context) {
     abort();
 }
-#endif // !defined(__USING_WASM_EXCEPTIONS__)
 
-using namespace __cxxabiv1;
+namespace __cxxabiv1 {
 
-//  Some utility routines are copied from cxa_exception.cpp
-static inline __cxa_exception*
-cxa_exception_from_thrown_object(void* thrown_object) {
-  return static_cast<__cxa_exception*>(thrown_object) - 1;
+//  Utility routines
+static
+inline
+__cxa_exception*
+cxa_exception_from_thrown_object(void* thrown_object)
```

Comment:
These utilities seem to be duplicated here and in the new utils file.

Reply:
Yeah, just like in cxa_exception they were originally borrowed from, they're static local helpers everywhere.